mediator/src/routes/patient.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+const patientController = require('../controllers/patient');
+
+router.post('/', async function(req, res) {
+  const patient = await patientController.createPatient(req);
+  res.send({status: 'success', patient});
```

Comment:
The response code always returns a 200 instead of a 201 and the response data type is a bit tricky

```ts
interface PatientHTTPResponse {
   status: "success",
   patient: {
     data: Patient/Error,
     status: number
   }
}
```
`createPatient` returns `patient` and the `status`, we can update the `req.status` with the `status` and only return `patient` back to the client.